ci/travis/dump-reports.sh

```diff
@@ -44,9 +44,9 @@ export -f dump_report
 # are copied into the scan-cmake-out directory by the build-docker.sh script.
 
 find "${BUILD_OUTPUT}" -name 'compat_report.html' \
-    -exec bash -c "dump_report '{}'" \; 2>/dev/null || \
+    -exec bash -c "dump_report '$1'" _ {} \; 2>/dev/null || \
```

Comment:
Don't the quotes have to be around the other way here (i.e., `bash -c 'dump_report "$1"' _ {}`) lest the `$1` gets expanded too early?

Reply:
You are correct. Good catch.